src/test/tcp/test_tcp.c

```diff
@@ -49,6 +61,37 @@ static void _mylog(const char* fileName, const int lineNum, const char* funcName
     fflush(stdout);
 }
 
+// create a System V queue with key based on a file queue name
+static int get_queue(const char* queuename) {
+    key_t key = ftok(queuename, 0);
+    assert_true_errno(key != -1);
+
+    int fd = msgget(key, IPC_CREAT | 0666);
+    assert_true_errno(fd != -1);
+
+    return fd;
+}
+
+static void queue_send_u16(const char* queuename, uint16_t i) {
+    int queue, r;
+    IntegerMessage msg = {QUEUE_MTYPE, i};
+
+    queue = get_queue(queuename);
+    r = msgsnd(queue, &msg, sizeof(msg.msg), IPC_NOWAIT);
+    assert_true_errno(r != -1);
+}
+
+static short queue_recv_u16(const char* queuename) {
+    int queue, r;
+    IntegerMessage msg;
+
+    queue = get_queue(queuename);
+    r = msgrcv(queue, &msg, sizeof(msg.msg), QUEUE_MTYPE, MSG_NOERROR | IPC_NOWAIT);
```

Comment:
We don't want `MSG_NOERROR` - we'd rather have an error than silent runcation.

We don't want `IPC_NOWAIT` - we'd rather wait for the writer to send than fail if this executes first.

Reply:
hum, I tried with other options but I think I should at least let `MSG_NOERROR`.. But, I don't think it's a big deal because if the message is not correct the server port will not be correct either and tests will fail. On the other hand, this will be more difficult to troubleshoot because the error will not be correctly located